app/Models/Store/Order.php

```diff
@@ -627,39 +627,37 @@ public static function cart($user)
             ->first();
     }
 
-    public function macroItemsQuantities(): \Closure
```

Comment:
this one doesn't seem to be used so I wonder if I should just delete it instead...?

Reply:
can probably remove it; I feel like it was for one of the order displays that's long gone